src/NHibernate.Test/Linq/ProjectionsTests.cs

```diff
@@ -294,7 +369,26 @@ public void ProjectKnownTypeWithCollection()
 			Assert.That(result.Count, Is.EqualTo(830));
 			Assert.That(result[0].ExpandedElement.OrderLines, Is.EquivalentTo(result[0].ProjectedProperty0));
 		}
-		
+
+		[Test]
+		public void ProjectKnownTypeWithCollectionWithFetch()
+		{
+			var query = from o in db.Orders.Fetch(x => x.OrderLines)
+			            select new ExpandedWrapper<Order, ISet<OrderLine>>
+			            {
+				            ExpandedElement = o,
+				            ProjectedProperty0 = o.OrderLines,
+				            Description = "OrderLine",
+				            ReferenceDescription = "OrderLine"
+			            };
+
+			var result = query.ToList();
+			Assert.That(result.Count, Is.EqualTo(830));
+			Assert.That(NHibernateUtil.IsInitialized(result[0].ExpandedElement.OrderLines), Is.True);
+			Assert.That(NHibernateUtil.IsInitialized(result[0].ProjectedProperty0), Is.True);
```

Comment:
Add these checks to other tests you've added above

Reply:
There is only one test that uses the `ExpandedWrapper`, but I can add the IsInitialized check to the other too.